op-verifier/README.md

```diff
@@ -63,6 +63,17 @@ The tests will require small modifications to work on public testnets; specifica
 
 - https://base-sepolia-gateway-worker.ens-cf.workers.dev/{sender}/{data}.json
 
+## OPOutputLookup
```

Comment:
Can you include the compilation parameters that were used here, so people can reproduce the build?

Reply:
Included

* Address: 0xbe3E115F5323E1e6fFd9746Cd6Aba5Ba7F2E6F8e
* Deterministic Deployment Proxy: 0x4e59b44847b379578588920cA78FbF26c0B4956C
* Salt: 0x0000000000000000000000000000000000000000000000000000000000000000
* Contract: lib/OPOutputLookup.sol:OPOutputLookup
* Solidity: 0.8.26
* Optimizer Runs: 200